extensions/cdc-postgres/src/main/java/com/hazelcast/jet/cdc/postgres/PostgresCdcSources.java

```diff
@@ -46,12 +46,20 @@ private PostgresCdcSources() {
      * Creates a CDC source that streams change data from a PostgreSQL database
      * to Hazelcast Jet.
      * <p>
-     * If Jet can't reach the database when it attempts to start the source or
-     * if it looses the connection to the database from an already running
-     * source, it throws an exception and terminate the execution of the job.
-     * This behaviour is not ideal, would be much better to try to reconnect,
-     * at least for a certain amount of time. Future versions will address the
-     * problem.
+     * <b>KNOWN ISSUE 1:</b> If Jet can't reach the database when it attempts to
+     * start the source or if it looses the connection to the database from an
+     * already running source, it throws an exception and terminate the
+     * execution of the job. This behaviour is not ideal, would be much better
+     * to try to reconnect, at least for a certain amount of time. Future
+     * versions will address the problem.
+     * <p>
+     * <b>KNOWN ISSUE 2:</b> The Debezium PostgreSQL Connector this source is
```

Comment:
`The Debezium PostgreSQL Connector this source is based on has data loss issues when ...` - this should be rewritten, seem like there are some left words after rewritting

Reply:
No, it's actually correct, barring some comas missing, but I'll rephrase it to a simpler form, because that will make it clearer.